.pre-commit-config.yaml

```diff
@@ -3,10 +3,11 @@ repos:
     hooks:
       - id: black
         name: black
-        entry: "dev/black.sh"
+        entry: black
```

Comment:
Currently, the black hook doesn't format python files. It just checks whether they are formatted. Once this PR is merged, the hook formats them.